MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -8,9 +8,11 @@ protocol FeedViewControllerDelegate {
 	func didRequestFeedRefresh()
 }
 
-public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView {
+public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView, FeedLoadErrorView {
 	var delegate: FeedViewControllerDelegate?
 
+	@IBOutlet var errorView: ErrorView!
```

Comment:
This property can be private
```suggestion
	@IBOutlet private var errorView: ErrorView!
```